avm/ptn/app/container-job-toolkit/tests/e2e/waf-aligned/dependencies.bicep

```diff
@@ -0,0 +1,42 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Log Analytics Workspace.')
+param lawName string
+
+@description('Required. The name of the Application Insights service.')
+param appInsightsName string
+
+@description('Required. The name of the User Assigned Identity.')
+param userIdentityName string
+
+resource law 'Microsoft.OperationalInsights/workspaces@2023-09-01' = {
+  name: lawName
+  location: location
+}
+
+module appInsights 'br/public:avm/res/insights/component:0.4.1' = {
```

Comment:
Why using a module instead of a resource block here? Just curious if there is a reason.

Reply:
I wanted to use the connectionString output from it